### PR TITLE
User follows

### PR DESCRIPTION
**Pull request by devton**
- [x] Add user friends endpoint to recommend friends to follow (starts with facebook friends)
- [x] Add user follow endpoint to handle with follow / unfollow users 
- [x] Add rpc to follow all friends at once
- [x] Add notification when a follow contribute with some project
- [x] Add notification when a follow publish a new project
- [x] Add user friends collection to a job
- [x] Add FB auth to an logged account
- [x] Check / Add trigger when payment has confirmed to broadcast a notification to followers (check if aleady sent using metadata:json on notifications) - not needed to be a trigger Unique index solves :)

### CODE REVIEW

File: app/views/catarse_bootstrap/user_notifier/mailer/follow_contributed_project_subject.text.slim [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- user_follow = @notification.user_follow
+- follow = user_follow.follow
+| #{follow.name} acabou de aopiar um projeto no #{CatarseSettings[:company_name]}
```
Comment (adrianob) [inline app/views/catarse_bootstrap/user_notifier/mailer/follow_contributed_project_subject.text.slim]: Typo

File: app/workers/fb_friend_collector_worker.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class FbFriendCollectorWorker
+  include Sidekiq::Worker
+  sidekiq_options retry: false
+
+  def perform(authorization_id)
+    if auth = Authorization.find(authorization_id) #yep is assign ;)
+      user_friends = auth.user.user_friends
+      last_friend = user_friends.order('id DESC').first
+
+      return if last_friend && last_friend.created_at > 24.hours.ago
```
Comment (adrianob) [inline app/workers/fb_friend_collector_worker.rb]: Shouldn't this be ```<  24.hours.ago```?

Comment (devton) [inline app/workers/fb_friend_collector_worker.rb]: nope :) we just rerun when the last friend as created in interval of 24 hours

File: app/views/catarse_bootstrap/projects/index.html.slim [hunk 3]
```diff
@@ -1,7 +1,7 @@
 - content_for :title, t('site.title')
 = render 'shared/header_big', overlayer: true
 
-#project-index-root[data-mithril="ProjectsHome"]
+#project-index-root[data-mithril="ProjectsHome" data-currentuserid=current_user.try(:id) data-hasfb="#{current_user.try(:has_fb_auth?)}"]
```
Comment (vicnicius) [inline app/views/catarse_bootstrap/projects/index.html.slim]: What do you think of adding those two fields to the data-user attribute that is already on the body tag?

**Pull request merged**